addons/sourcemod/scripting/vsh/bosses/boss_whiteranger.sp

```diff
@@ -0,0 +1,380 @@
+			case VSHSound_Rage: strcopy(sSound, length, g_strUberRangerRage[GetRandomInt(0,sizeof(g_strUberRangerRage)-1)]);
+			case VSHSound_Lastman: strcopy(sSound, length, g_strUberRangerLastMan[GetRandomInt(0,sizeof(g_strUberRangerLastMan)-1)]);
+			case VSHSound_Backstab: strcopy(sSound, length, g_strUberRangerBackStabbed[GetRandomInt(0,sizeof(g_strUberRangerBackStabbed)-1)]);
+		}
+	}
+	
+	public void GetSoundAbility(char[] sSound, int length, const char[] sType)
+	{
+		if (strcmp(sType, "CBraveJump") == 0)
+			strcopy(sSound, length, g_strUberRangerJump[GetRandomInt(0,sizeof(g_strUberRangerJump)-1)]);
+	}
+	
+	public void GetMusicInfo(char[] sSound, int length, float &time)
+	{
+		strcopy(sSound, length, RANGER_THEME);
+		time = 235.0;
+	}
+	
+	public void Precache()
+	{
+		PrecacheModel(RANGER_MODEL);
+		
+		g_iUberRangerPrussianPickelhaube = PrecacheModel("models/player/items/medic/medic_helmet.mdl");
+		g_iUberRangerBlightedBeak = PrecacheModel("models/player/items/medic/medic_blighted_beak.mdl");
+		
+		PrepareSound(RANGER_THEME);
+		PrecacheSound(RANGER_RAGESOUND);
+		
+		for (int i = 0; i < sizeof(g_strUberRangerRoundStart); i++) PrecacheSound(g_strUberRangerRoundStart[i]);
+		for (int i = 0; i < sizeof(g_strUberRangerWin); i++) PrecacheSound(g_strUberRangerWin[i]);
+		for (int i = 0; i < sizeof(g_strUberRangerLose); i++) PrecacheSound(g_strUberRangerLose[i]);
+		for (int i = 0; i < sizeof(g_strUberRangerLastMan); i++) PrecacheSound(g_strUberRangerLastMan[i]);
+		for (int i = 0; i < sizeof(g_strUberRangerBackStabbed); i++) PrecacheSound(g_strUberRangerBackStabbed[i]);
+		for (int i = 0; i < sizeof(g_strUberRangerJump); i++) PrecacheSound(g_strUberRangerJump[i]);
+		
+		AddFileToDownloadsTable("materials/models/player/boss/uber_ranger/uberranger_backpack.vmt");
+		AddFileToDownloadsTable("materials/models/player/boss/uber_ranger/uberranger_backpack.vtf");
+		AddFileToDownloadsTable("materials/models/player/boss/uber_ranger/uberranger_body.vmt");
+		AddFileToDownloadsTable("materials/models/player/boss/uber_ranger/uberranger_body.vtf");
+		AddFileToDownloadsTable("materials/models/player/boss/uber_ranger/uberranger_head.vmt");
+		AddFileToDownloadsTable("materials/models/player/boss/uber_ranger/uberranger_head.vtf");
+		
+		AddFileToDownloadsTable("models/player/vsh_rewrite/uber_ranger/uber_ranger.mdl");
+		AddFileToDownloadsTable("models/player/vsh_rewrite/uber_ranger/uber_ranger.sw.vtx");
+		AddFileToDownloadsTable("models/player/vsh_rewrite/uber_ranger/uber_ranger.vvd");
+		AddFileToDownloadsTable("models/player/vsh_rewrite/uber_ranger/uber_ranger.phy");
+		AddFileToDownloadsTable("models/player/vsh_rewrite/uber_ranger/uber_ranger.dx80.vtx");
+		AddFileToDownloadsTable("models/player/vsh_rewrite/uber_ranger/uber_ranger.dx90.vtx");
+	}
+	
+	public void Destroy()
+	{
+		SetEntityRenderColor(this.iClient, 255, 255, 255, 255);
+	}
```

Comment:
Missed that to remove, if going for attribute way to color cosmetics

Reply:
it would go from 230 230 230 to 255 255 255

Reply:
You are right, i missed that `SetEntityRenderColor` is still used for boss model